documentation/modules/exploit/windows/http/trend_micro_officescan_widget_exec.md

```diff
@@ -0,0 +1,61 @@
+## Vulnerable Application
+
+This module exploits the authentication bypass and command injection vulnerability together. Unauthenticated users can execute a terminal command under the context of the web server user.
+
+Trend Micro Officescan product have widget feature which is implemented with PHP. Talker.php takes ack and hash parameter but don't validate these values, which leads to an authentication bypass for widget. Proxy.php files under the mod TMCSS folder takes multiple parameter but the process does not properly validate a user-supplied string before using it to execute a system call. Due to combination of these vulnerabilities, unauthenticated users can execute a terminal command under the context of the web server user.
```

Comment:
**The** Trend Micro Office**S**can product **has a** widget feature which is implemented with PHP. Talker.php takes ack and hash parameter**s** but **doesn't** validate these values, which leads to an authentication bypass for **the** widget. Proxy.php files under the mod TMCSS folder **take** multiple **parameters** but the process does not properly validate a user-supplied string before using it to execute a system call. Due to combination of these vulnerabilities, unauthenticated users can execute a terminal command under the context of the web server user.